build.gradle

```diff
@@ -181,9 +183,21 @@ tasks.withType(io.gitlab.arturbosch.detekt.Detekt.class) {
     exclude("**/tmp/**")
 }
 
-task listRepositories {
+tasks.register("listRepositories") {
     doLast {
         println "Repositories:"
         project.repositories.each { println "Name: " + it.name + "; url: " + it.url }
    }
 }
+
+tasks.register("writeGithubTestDetails", GithubDetailsTask) {
```

Comment:
I'd like a different name, maybe "githubDetails" instead of "writeGithubTestDetails"?